conductor-client/src/main/kotlin/com/openlattice/data/storage/BinaryDataWithMetadata.kt

```diff
@@ -0,0 +1,67 @@
+package com.openlattice.data.storage
+
+import com.google.common.base.Preconditions
+import java.util.*
+
+/**
+ *
+ * @author Matthew Tamayo-Rios &lt;matthew@openlattice.com&gt;
+ */
+data class BinaryDataWithMetadata(
```

Comment:
Wouldn't `BinaryObjectWithMetadata` make more sense?

Reply:
IMO they're pretty equally sensible but i don't really care so I'll hit that refactor button.